packages/cli/test/commands/deploy.test.ts

```diff
@@ -27,7 +29,7 @@ describe('deploy', () => {
         const fakeDeployer = fake()
         replace(environment, 'currentEnvironment', fake.returns('test-env'))
```

Comment:
I think that you should mock `pruneDependencies` and `reinstallDependencies` here to make sure they're not actually running the `exec`s